packages/forms/src/UIForm/fields/MultiSelectTag/__snapshots__/MultiSelectTag.component.test.js.snap

```diff
@@ -40,7 +39,7 @@ exports[`MultiSelectTag field should render MultiSelectTag 1`] = `
       position="left"
       readOnly={false}
       searching={false}
-      searchingText="Searching for matches..."
+      searchingText="Searching for matches…"
```

Comment:
The changes in snapshot are weird. Have you compiled all the repo ?

Reply:
fixed, component was not up to date